api/views/CloudRecordSummaryView.py

```diff
@@ -174,6 +141,7 @@ def get(self, request, format=None):
 
     def _parse_query_parameters(self, request):
         """Parse expected query parameters from the given HTTP request."""
+        logger = logging.getLogger(__name__)
```

Comment:
Later on it says to not get logger at the module level, but doing it at the class level in the `_init_` seems fine.